### ISSUE TITLE
Package installs as 'pyECCArithmetic' on macOS and import fails

### ISSUE THREAD

**Issue by bjk17** [Kavakuo/PyECCArithmetic#1]
*Note the lower-case 'p' at the start of the package name.* This leads to the `from pyECCArithmetic.x import Y` statements in `__init__.py` to fail. The error manifests itself as follows:

```zsh
➜  ~ python --version
Python 3.7.3
➜  ~ virtualenv --version
16.4.1
➜  ~        
➜  ~ virtualenv /tmp/venv
Using base prefix '/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7'
New python executable in /tmp/venv/bin/python3.7
Also creating executable in /tmp/venv/bin/python
Installing setuptools, pip, wheel...
done.
➜  ~ source /tmp/venv/bin/activate
(venv) ➜  ~ pip install PyECCArithmetic
Collecting PyECCArithmetic
  Using cached https://files.pythonhosted.org/packages/76/39/ecf93d5bc45d70f77084d02e9eb8aba5acf73b61aebac55a67265c3dc7a1/PyECCArithmetic-1.0.1-py3-none-any.whl
Installing collected packages: PyECCArithmetic
Successfully installed PyECCArithmetic-1.0.1
(venv) ➜  ~ pip list
Package         Version
--------------- -------
pip             19.2.1 
PyECCArithmetic 1.0.1  
setuptools      41.0.1 
wheel           0.33.4 
(venv) ➜  ~
(venv) ➜  ~ echo "Upper-case 'P' as per documentation"
Upper-case 'P' as per documentation
(venv) ➜  ~ python -c "import PyECCArithmetic; print('Success?')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'PyECCArithmetic'
(venv) ➜  ~ echo "Lower-case 'p' like the package was installed"
Lower-case 'p' like the package was installed
(venv) ➜  ~ python -c "import pyECCArithmetic; print('Success?')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/venv/lib/python3.7/site-packages/pyECCArithmetic/__init__.py", line 4, in <module>
    from PyECCArithmetic.point import Point
ModuleNotFoundError: No module named 'PyECCArithmetic'
(venv) ➜  ~ l /tmp/venv/lib/python3.7/site-packages/               
total 8
drwxr-xr-x  13 bjarni  wheel   416B Jul 24 17:32 .
drwxr-xr-x  59 bjarni  wheel   1.8K Jul 24 17:32 ..
drwxr-xr-x   8 bjarni  wheel   256B Jul 24 17:32 PyECCArithmetic-1.0.1.dist-info
drwxr-xr-x   3 bjarni  wheel    96B Jul 24 17:32 __pycache__
-rw-r--r--   1 bjarni  wheel   126B Jul 24 17:32 easy_install.py
drwxr-xr-x   7 bjarni  wheel   224B Jul 24 17:32 pip
drwxr-xr-x   9 bjarni  wheel   288B Jul 24 17:32 pip-19.2.1.dist-info
drwxr-xr-x   7 bjarni  wheel   224B Jul 24 17:32 pkg_resources
drwxr-xr-x   7 bjarni  wheel   224B Jul 24 17:32 pyECCArithmetic
drwxr-xr-x  42 bjarni  wheel   1.3K Jul 24 17:32 setuptools
drwxr-xr-x  11 bjarni  wheel   352B Jul 24 17:32 setuptools-41.0.1.dist-info
drwxr-xr-x  12 bjarni  wheel   384B Jul 24 17:32 wheel
drwxr-xr-x   9 bjarni  wheel   288B Jul 24 17:32 wheel-0.33.4.dist-info
```

By manually editing the import statements in the `__init__.py` file I'm able to use the library:

```zsh
(venv) ➜  ~ sed -i -e 's/PyECCArithmetic/pyECCArithmetic/g' /tmp/venv/lib/python3.7/site-packages/pyECCArithmetic/__init__.py
(venv) ➜  ~ cat /tmp/venv/lib/python3.7/site-packages/pyECCArithmetic/__init__.py
# coding=utf-8
name = "pyECCArithmetic"

from pyECCArithmetic.point import Point
from pyECCArithmetic.curve import Curve
from pyECCArithmetic.error import PointsOnDifferentCurveError


__all__ = ['Point', 'Curve', 'PointsOnDifferentCurveError']
(venv) ➜  ~ python -c "import pyECCArithmetic; print('Success?')"
Success?
```

**Comment by bjk17** [comment]
Renaming `pyECCArithmetic` directory to `PyECCArithmetic` in `/tmp/venv/lib/python3.7/site-packages/` instead of modifying the `__init__.py` file also works for me.

```zsh
(venv) ➜  ~ mv /tmp/venv/lib/python3.7/site-packages/pyECCArithmetic /tmp/venv/lib/python3.7/site-packages/PyECCArithmetic
(venv) ➜  ~ python -c "import PyECCArithmetic; print('Success?')"
Success?
```

**Comment by Kavakuo** [comment]
Just pushed a new version, the issue should be fixed with version 1.0.2. Could you be so kind and try to verify this?
I don't really know what went wrong before. It seems that something messed up the build process...

**Comment by bjk17** [comment]
I can verify that version 1.0.2 installs and imports OK on my macOS machine, thank you very much. The directory under `site-packages` has the upper-case "P" so everything works according to your documentation.
